elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -164,12 +165,15 @@ default void preCommit() {
             String relationName,
             Class<T> relationClass,
             EntityDictionary dictionary,
-            Set<Predicate> filters
+            FilterScope filterScope
```

Comment:
Note: 
Passing filterScope here because in cases of getting a relation from another dataStore, this is expected to call
`filterScope.getRequestScope().getTransaction().loadObjects(relationClass, filterScope)` where `filterScope.getRequestScope().getTransaction()` must be a MultiplexTransaction (cuz otherwise you won't be able to cross datastores).